### PR TITLE
Add Hirochika Matsumoto to wg-prioritization

### PR DESCRIPTION
**Pull request by apiraino**
Discussed on [Zulip](https://rust-lang.zulipchat.com/#narrow/stream/227806-t-compiler.2Fwg-prioritization/topic/Hirochika.20Matsumoto.20wants.20to.20join)

@hkmatsumoto also review if I picked the correct usernames/ID, thanks.

cc: @spastorino

### CODE REVIEW

**Review by hkmatsumoto — APPROVED** [review]
Both names and ids are correct, thanks!
For the file name, let's rename `hirochikamatsumoto.yml` to `hkmatsumoto.yml`, because It looks like people tend to match their file names and GitHub ids. That will hopefully make CI pass too.

**Comment by camelid** [comment]
@apiraino You need to name the file `.toml`, not `.yml` :)

**Comment by camelid** [comment]
@apiraino It's probably best to just run `cargo run add-person <github-username>` since it will populate the fields correctly for you.

File: teams/wg-prioritization.toml [hunk 1]
```diff
@@ -27,6 +27,7 @@ members = [
     "Stupremee",
     "tamuhey",
     "wesleywiser",
+    "hkmatsumoto"
```
Comment (camelid) [inline teams/wg-prioritization.toml:30]: Please also add a trailing comma to avoid future diff noise.

**Comment by apiraino** [comment]
thanks @camelid , force pushed based on your suggestions

**Review by spastorino — APPROVED** [review]

**Comment by apiraino** [comment]
cc @Mark-Simulacrum

**Review by pietroalbini — APPROVED** [review]

**Pull request merged**